### ISSUE TITLE
OpenShift deployment failed on latest master due to syntax error while resolving openshift.common.is_atomic

### ISSUE THREAD

**Issue by smarterclayton** [openshift/openshift-ansible#5034]
```
TASK [/usr/share/ansible/openshift-ansible/roles/os_firewall : Assert - Do not use firewalld on Atomic Host] ***
task path: /usr/share/ansible/openshift-ansible/roles/os_firewall/tasks/main.yml:2
Tuesday 08 August 2017  19:05:32 +0000 (0:00:00.349)       0:04:06.248 ******** 
fatal: [ci-primg360-ig-m-40q8]: FAILED! => {
    "failed": true, 
    "msg": "The conditional check 'openshift.common.is_atomic | bool' failed. The error was: error while evaluating conditional (openshift.common.is_atomic | bool): 'openshift' is undefined\n\nThe error appears to have been in '/usr/share/ansible/openshift-ansible/roles/os_firewall/tasks/main.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Assert - Do not use firewalld on Atomic Host\n  ^ here\n"
}
```

https://ci.openshift.redhat.com/jenkins/job/zz_origin_gce_image/360/consoleText

**Comment by sdodson** [comment]
origin-gce needs to add playbooks/common/openshift-cluster/std_include.yml somewhere. @kwoodson can you take a look at https://github.com/openshift/origin-gce/blob/master/playbooks/configure.yaml and help @smarterclayton out?

**Comment by sdodson** [comment]
We probably need to verify that the ref-arches aren't totally wrecked now too. CC: @cooktheryan

**Comment by cooktheryan** [comment]
@sdodson yikes. yes thanks for the heads up

**Comment by sdodson** [comment]
Basically if you're calling directly into a role that depended on openshift_facts you're probably going to have a bad time. If you're only calling playbooks/byo/* you should be fine.

For this particular problem we can likely just get rid of anything openshift_facts related in os_firewall.

**Comment by smarterclayton** [comment]
Fixed in openshift/origin-gce#41, until such a time as os_firewall is updated.  Lowering priority and assigning to myself to go fix.
